chalicelib_fourfront/checks/wfr_encode_checks.py

```diff
@@ -158,15 +169,16 @@ def chipseq_status(connection, **kwargs):
                 for merge_case in input_list:
                     merge_enum += 1
```

Comment:
It may not matter but it could be confusing if this is incremented even though there is nothing in 'merge_case' - may want to move this after the if?